docs/triggerservice_module.rst

```diff
@@ -4,6 +4,14 @@ triggerservice Module
 
 .. currentmodule:: mwa_trigger.triggerservice
 
+trigger_mwa Function
+--------------------
+.. autofunction:: trigger_mwa
```

Comment:
these `autofunction` directives don't seem to generate any content.

Reply:
There was an issue with the installation of the dependencies which I've fixed now (you may need to use crtl f5 to see the updates)